addons/mail/wizard/discuss/discuss_channel_owner_edit.py

```diff
@@ -0,0 +1,54 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api
+
+
+class DiscussChannelOwnerEdit(models.TransientModel):
+    _name = 'discuss.channel.owner.edit'
```

Comment:
Couldn't we do a first version of the PR without this extra code now that we can transfer differently?